tools/local-storage/src/storage.ts

```diff
@@ -0,0 +1,29 @@
+import storage from 'local-storage-fallback'
```

Comment:
I don't really understand why we need this file, why not just shim `window.localStorage`?

Reply:
Storage options can be controlled by the user agent. This tries to use the following mechanisms

```
localStorage, sessionStorage, cookies, memory
```

Reply:
I meant the following: `window.localStorage = localStorageFallback`

Reply:
Gotcha. I went with this approach because it's a library and can avoid a conflict if the consuming app doesn't want to shim it.